docs/proposals/20200330-spot-instances.md

```diff
@@ -249,6 +250,40 @@ The Node will transition to an unready state which would be detected by a Machin
 though there may be some delay depending on the configuration of the MachineHealthCheck.
 In the future, a termination handler could trigger the Machine to be deleted sooner.
 
+
+
+
+### 'Interruptible' label
+
+In order to deploy the termination handler, we'll need to create a DaemonSet that runs it on each spot instance node.
+
+Having `"cluster.x-k8s.io/interruptible"` label on Nodes that run on interruptible instances should help us with it.
+
+Based on the discussion here https://github.com/kubernetes-sigs/cluster-api/pull/3668 ([1](https://github.com/kubernetes-sigs/cluster-api/pull/3668#issuecomment-696143653), [2](https://github.com/kubernetes-sigs/cluster-api/pull/3668#issuecomment-696862994).) we can do following: 
+1. User creates InfraMachine with whatever spec field(s) are required for that provider to indicate it's interruptible.
+2. Infra provider sets InfraMachine.status.interruptible=true
+3. Machine looks at the infra machine's status (for interruptible) and adds a label to the node.
```

Comment:
This might be clearer? I feel like we can probably combine 3 + 4, WDYT? 
```suggestion
3. Machine controller looks at InfraMachine.status.interruptible and ensures a label is set on the node if it is true.
```